### PR TITLE
Fix broken dictcomp test on Python 3.8

### PR DESCRIPTION
**Pull request by malthe**
See https://github.com/python/cpython/commit/c8a35417db8853a253517a3e5190e174075c6384

### CODE REVIEW

**Comment by davisagli** [comment]
It looks like the Python 2.7 appveyor build needs either a newer pip or an older pytest.

File: tests/transformer/test_dict_comprehension.py [hunk 1]
```diff
@@ -24,10 +24,23 @@ def test_dict_comprehension_with_attrs(mocker):
...
-        mocker.call(z[0], 'k'),
-        mocker.call(z[1], 'k'),
-        mocker.call(z[1], 'v'),
-        mocker.call(z[1], 'k')
-    ])
+
+    calls = [mocker.call(seq, 'z')]
+
+    # Note: Order changed in PEP 572, starting with Python 3.8.
+    if version_info.major == 3 and version_info.minor >= 8:
```
Comment (icemac) [inline tests/transformer/test_dict_comprehension.py]: We already have a constant for this comparison, see

https://github.com/zopefoundation/RestrictedPython/blob/49fe489c2bd08c6451bdc1e8a3df716edc4faca6/src/RestrictedPython/_compat.py#L12

Comment (malthe) [reply tests/transformer/test_dict_comprehension.py]: Fixed

**Comment by icemac** [comment]
Thank you very much!

**Pull request merged**